metadata.json

```diff
@@ -7,6 +7,12 @@
   "source": "https://github.com/dalen/puppet-dnsquery.git",
   "project_page": "https://github.com/dalen/puppet-dnsquery",
   "issues_url": "https://github.com/dalen/puppet-dnsquery/issues",
+  "requirements": [
+    {
+      "name": "puppet",
+      "version_requirement": ">= 6.0.0 < 8.0.0"
```

Comment:
please go to 6.1.0 as minimal version. That version has support for detecting systemd services that require a reload.

Reply:
done